pyproject.toml

```diff
@@ -18,7 +18,7 @@ dependencies = [  # runtime deps  # https://packaging.python.org/en/latest/guide
   # GUIDANCE: only add things that this project imports directly
   # GUIDANCE: only set lower version bounds
   # "awx_plugins.base_interface.api",  # keep `__init__.py` empty
-  "awx_plugins.interfaces",  # standard interface declarations for AWX plugins
+  "awx_plugins.interfaces @git+https://github.com/ansible/awx_plugins.interfaces.git@ad4a965",  # standard interface declarations for AWX plugins
```

Comment:
Do we think what is there is good enough already? I think the reason I put this here was because when I was trying to test things, it was screaming about missing something so I dropped it in here.

Reply:
Well, we might need to do pinning in that constraint file later on, but it shouldn't be in this PR, I think.

Locally, you might've just hit the cache. You can pass `-r` to tox to force it to recreate its venvs.

I've hit caching similar issues with `pre-commit`, which is why I've started pinning the interfaces there. It can be addressed locally but in the `pre-commit.ci` service, such cache is immortal. So yes, we do need to handle this in general, where necessary. If we get to version the libs properly, this will become easier.

Reply:
ahh gotcha, ty for the `-r`

Reply:
@thedoubl3j Note that `-r` will only clean tox's own venvs. If you want to clean what the `pre-commit` tool manages, you could try `tox e -e pre-commit -- pytest -Im pre_commit clean` and it'll clear the userspace venvs it manages.

Reply:
```suggestion
  "awx_plugins.interfaces",  # standard interface declarations for AWX plugins
```